hugolib/page.go

```diff
@@ -168,30 +168,55 @@ func (p *Page) Param(key interface{}) (interface{}, error) {
 	return p.Site.Params[keyStr], nil
 }
 
+// Author returns the first listed author for a page
 func (p *Page) Author() Author {
-	authors := p.Authors()
+	authors := p.getAuthors()
```

Comment:
See comment below: If we want to pick 1 author as THE author, we must define what that is. I think the first one in a well defined sorted list (see below) would make sense.